packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,149 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/44189.
+  - title: 'Rename to dependOnInheritedElement'
+    date: 2020-12-23
+    element:
+      uris: [ 'material.dart', 'cupertino.dart', 'widgets.dart' ]
```

Comment:
@bwilkerson I may not be using this correctly. I wrote tests to check each of these uris, but only one of them passed (`widgets.dart`) Am I missing something?

Reply:
Hm, I am thinking this is my error, https://github.com/flutter/flutter/pull/72901 works fine and is very similar.

Reply:
Found it. :) My mistake indeed.